npm/react/cypress/component/advanced/renderless/mouse.cy.jsx

```diff
@@ -23,7 +23,8 @@ describe('Renderless component', () => {
       expect(onMoved).to.have.been.calledWith(true)
     })
 
-    unmount()
+    // mount something else to trigger unmount
+    mount(<div>Test Component</div>)
```

Comment:
Just a random thought, should we use our suggested recommendation for unmounting post v11 in this test?

`cy.then(() => ReactDom.unmountComponentAtNode(getContainerEl()))`